python_modules/dagster/dagster/_core/instance/ref.py

```diff
@@ -418,11 +418,20 @@ def from_dir(
             config_value, "scheduler", defaults["scheduler"]
         )
 
-        run_coordinator_data = configurable_class_data_or_default(
-            config_value,
-            "run_coordinator",
-            defaults["run_coordinator"],
-        )
+        if config_value.get("run_queue"):
```

Comment:
could there be some confusing case since

```
run_queue: {}
```

doesn't enable the daemon but 

```
run_queue:
  tag_concurrency_limits: []
```

does? I think that seems fine

Reply:
I think its probably fine to have empty mean disabled, yeah